components/playwright/README.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+[Playwright](https://pptr.dev/) is a Node.js library which provides a high-level API to control Chrome/Chromium over the DevTools Protocol. Playwright runs in headless mode on Chromium on Pipedream.
+
+Using Playwright you can perform tasks including:
+
+* Screenshots and PDFs: Capture web page visuals.
```

Comment:
I would explicitly split out PDFs into a new section like:

> * Processing PDFs: parse and scan PDFs

Reply:
Applied to puppeeteer's README as well